src/main/java/reactor/netty/tcp/TcpResources.java

```diff
@@ -210,6 +210,7 @@ public void disposeWhen(@NonNull SocketAddress address) {
 	}
 
 	@Override
+	@SuppressWarnings("deprecation")
```

Comment:
is the method going to stay in `TcpResources` after being removed from `LoopResources` interface? If not I'd say suppressing warnings here don't make much sense

Reply:
(also valid for the 2 occurrences below)

Reply:
they will be removed in master
and it is suppressed because otherwise `errorprone` plugin complains about it

```
/.../reactor-netty/src/main/java/reactor/netty/tcp/TcpResources.java:225: warning: [deprecation] onDatagramChannel(EventLoopGroup) in LoopResources has been deprecated
        public Class<? extends DatagramChannel> onDatagramChannel(EventLoopGroup group) {
                                                ^
/.../reactor-netty/src/main/java/reactor/netty/tcp/TcpResources.java:226: warning: [deprecation] onDatagramChannel(EventLoopGroup) in LoopResources has been deprecated
                return defaultLoops.onDatagramChannel(group);
                                   ^
```